variable.tf

```diff
@@ -0,0 +1,17 @@
+variable "service_name" {
+  description = "The service enpoint name, should be in the format com.amazonaws.<region>.<service>"
+}
+
+variable "endpoint_type" {
+  description = "Interface or Gateway (gateway only valid for s3 and dynamodb)"
+}
+
+variable "tags" {
+  description = "Map of tags to be added to the set of default tags"
+  type        = map(string)
+  default     = {}
+}
```

Comment:
I still think we should remove this variable

Reply:
done